unikernel.ml

```diff
@@ -2105,14 +2082,10 @@ struct
                                    root policy: %s"
                                   err);
                             Middleware.http_response reqd
-                              ~data:
-                                (`String
-                                   ("error with root policy: "
-                                  ^ String.escaped err))
+                              ~data:(`String ("error with root policy: " ^ err))
                               `Internal_server_error)
                     | Error (`Msg err) ->
-                        Middleware.http_response reqd
-                          ~data:(`String (String.escaped err))
+                        Middleware.http_response reqd ~data:(`String err)
                           `Bad_request)
```

Comment:
here I'm a bit unsure -- so Albatross_json.policy_of_json in the error case uses `Yojson.Basic.to_string` -- and I'm not sure whether this is safe (in terms of injection attacks).